pkg/initialize/run.go

```diff
@@ -64,6 +65,12 @@ func Run(isScanCmd bool) error {
 
 // DownloadPolicies clones the policies to a local folder
 func DownloadPolicies() error {
+
+	policyBasePath := config.GetPolicyBasePath()
```

Comment:
instead of assigning new variables, we can use values directly?

Reply:
changes made where feasible